src/app/units/states/portfolios/portfolios.scss

```diff
@@ -10,3 +10,18 @@
     }
   }
 }
+
+.grade-icon {
```

Comment:
Good eye. This is for the 'All' button that was using the styling from old grade-icon scss. Since migrating, it doesn't have access to that styling which is why I duplicated it. Is there a better way to do this? 
<img width="268" alt="image" src="https://user-images.githubusercontent.com/49971210/167034481-b8f256ba-4a04-49ec-a3f3-35bc1b75103c.png">